hazyCalculator.js

```diff
@@ -3,16 +3,17 @@ function isSkippedValue(value) {
 }
 
 function isNumericValue(value) {
-  return !isNaN(value)
+  return !isNaN(value) && value != ''
```

Comment:
Careful to always use strict equality (`===` and `!==`) instead of loose equality (`==` and `!=`)